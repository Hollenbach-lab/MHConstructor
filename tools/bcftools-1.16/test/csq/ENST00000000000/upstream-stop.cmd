{bin}/bcftools csq -p a -f ENST00000000000.fa -g ENST00000000000.gff upstream-stop.vcf  | {bin}/bcftools query -f'[%POS\t%REF\t%ALT\t%SAMPLE\t%GT\t%TBCSQ\n]' | ../sort-csq -q
