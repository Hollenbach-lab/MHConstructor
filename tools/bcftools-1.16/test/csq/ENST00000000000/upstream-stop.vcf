##fileformat=VCFv4.2
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=I,length=15072434>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	aaa	bbb
I	6723	.	A	T	.	PASS	.	GT	1/1	0/0
I	6725	.	G	A	.	PASS	.	GT	1/1	0/0
I	7395	.	G	T	.	PASS	.	GT	1/1	1/1
