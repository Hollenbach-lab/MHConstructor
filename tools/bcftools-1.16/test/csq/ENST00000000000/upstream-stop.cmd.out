6723	A	T	aaa	1/1	stop_gained|WBGene00016561|C41D11.1e.1|protein_coding|+|49K>49*|6723A>T+6725G>A	stop_gained|WBGene00016561|C41D11.1e.1|protein_coding|+|49K>49*|6723A>T+6725G>A
6725	G	A	aaa	1/1	@6723	@6723
7395	G	T	aaa	1/1	*missense|WBGene00016561|C41D11.1e.1|protein_coding|+|259D>259Y|7395G>T	*missense|WBGene00016561|C41D11.1e.1|protein_coding|+|259D>259Y|7395G>T
7395	G	T	bbb	1/1	missense|WBGene00016561|C41D11.1e.1|protein_coding|+|259D>259Y|7395G>T	missense|WBGene00016561|C41D11.1e.1|protein_coding|+|259D>259Y|7395G>T
